<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureMentSettingDialog</class>
 <widget class="QDialog" name="MeasureMentSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,0">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">color: rgb(143, 143, 71);
font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="title">
      <string>GroupOne</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>706</width>
        <height>115</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>OverLap1:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>index:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G1index1"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G1offset1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>OverLap2:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>index:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G1index2"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G1offset2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>OverLap3:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>index:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G1index3"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G1offset3"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(143, 143, 71);
font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="title">
      <string>GroupTwo</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>706</width>
        <height>115</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>OverLap1:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>index:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G2index1"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G2offset1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>OverLap2:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>index:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G2index2"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G2offset2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>OverLap3:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>index:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G2index3"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>offset:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_G2offset3"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_saveStingBtn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;
background-color: rgb(138, 138, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Save Setting</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_applyBtn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;
background-color: rgb(0, 144, 70);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>198</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
